test/snapp_auction.js

```diff
@@ -520,4 +521,31 @@ contract("SnappAuction", async (accounts) => {
       assert.equal(currentAuction.size, 2)
     })
   })
+
+  describe("Larger Test Cases", () => {
+    it("Fill order batch", async () => {
+      const instance = await SnappAuction.new()
+      await setupEnvironment(MintableERC20, instance, token_owner, [user_1], 2)
+      const order = encodeOrder(0, 1, 2, 3)
+      const maxAuctionSize = (await instance.AUCTION_BATCH_SIZE.call()).toNumber()
+      const orders = Array(maxAuctionSize).fill(order)
+      const partitionedOrders = splitArray(orders, 100)
+      let concatenated_orders
```

Comment:
nit: I'd declare inside the forEach function.

Reply:
I didn't realize I could do that or else I would have... Example please.

Reply:
```js
await partitionedOrders.forEach(async function(partition) {  // should be about 10
        const concatenated_orders = Buffer.concat(partition)
        await instance.placeSellOrders(concatenated_orders, { from: user_1 })
}
```